test/models.test.js

```diff
@@ -50,10 +52,74 @@ describe('collections', function () {
 
 describe('models', function() {
   describe('centre', function() {
+    beforeEach(function() {
+      var io;
+
+      this.server = new socket();
+      this.fakeio = function(server) {
+        return function() {
+          return server.socketClient;
+        }
+      };
+
+      io = sailsIOClient(this.fakeio(this.server));
+      io.sails.autoConnect = false;
+
+      this.socket0 = io.sails.connect();
+    });
     it('should be instantiated', function() {
       var model = new models.Centre();
 
       expect(model).not.to.be(undefined);
```

Comment:
huh? no, there should be just one expect per `it`, and it should be what the `it()` returns which allows the expects to bubble up through the reporting

```javascript
it('should be instantiated', function() {
  var model = new models.Centre();
  return expect(model).to.be.ok();
});
```